web-app/react-app/components/AuthPage/index.jsx

```diff
@@ -0,0 +1,51 @@
+// @flow
+
+import React from 'react'
+import { BrowserRouter, Route, Redirect, Switch } from 'react-router-dom'
+import createLink from '../../utils/createLink'
+
+import 'bootstrap/dist/css/bootstrap.min.css'
+import '@streamr/streamr-layout/css'
+import '@streamr/streamr-layout/pcss'
+import 'react-select/dist/react-select.css'
+
+import LoginPage from './pages/LoginPage'
+import RegisterPage from './pages/RegisterPage'
+import SignupPage from './pages/SignupPage'
+import ForgotPasswordPage from './pages/ForgotPasswordPage'
+import ResetPasswordPage from './pages/ResetPasswordPage'
+import GoogleAnalyticsTracker from '../../components/GoogleAnalyticsTracker'
+import styles from './authPage.pcss'
+import isProduction from '../../utils/isProduction'
+
+const basename = createLink('/').replace(window.location.origin, '')
+
+const AuthPage = () => (
+    <BrowserRouter basename={basename}>
+        <div className={styles.authPage}>
+            <section className={styles.content}>
+                <a href="https://www.streamr.com" className={styles.logo}>
+                    <img src={createLink('static/images/streamr-logo.svg')} alt="Streamr logo" />
+                </a>
+                <div className={styles.panel}>
+                    <Switch>
+                        <Route exact path="/login/auth" component={LoginPage} />
+                        <Route exact path="/register/register" component={RegisterPage} />
+                        <Route exact path="/register/signup" component={SignupPage} />
+                        <Route exact path="/register/forgotPassword" component={ForgotPasswordPage} />
+                        <Route exact path="/register/resetPassword" component={ResetPasswordPage} />
+                        <Redirect from="/register" to="/register/signup" />
+                        <Redirect from="/login" to="/login/auth" />
+                        <Route path="/" component={() => '404'} />
```

Comment:
maybe we could route back to the `/login/auth` page instead of having a dedicated login 404

<img width="669" alt="screen shot 2018-07-20 at 9 08 32" src="https://user-images.githubusercontent.com/1593398/42994879-0724fb7c-8c18-11e8-90bb-215a4e58c521.png">